photon-icons/photon-icons.json

```diff
@@ -0,0 +1,36 @@
+{
+  "license": "This Source Code Form is subject to the terms of the Mozilla Public License, version 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.",
+  "fills": [
```

Comment:
Should we list all the colors above everything or list specific colors inside specific icons? Is there a way to import/reference the colors tokens?

Reply:
I think we can just use the color token names, and do the linking in code.